Sources/Ethereum/Ethereum.swift

```diff
@@ -20,10 +20,6 @@ open class Ethereum: Blockchain {
         return .ethereum
     }
 
-    open override var coinPurpose: Purpose {
-        return .bip44
```

Comment:
I would keep it here to make sure it’s correct purpose to use with ethereum

Reply:
default is .bip44 and we have coinPurpose tests